src/main/java/org/prebid/server/bidder/UsersyncInfoBuilder.java

```diff
@@ -0,0 +1,76 @@
+package org.prebid.server.bidder;
+
+import org.apache.commons.lang3.ObjectUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.prebid.server.privacy.model.Privacy;
+import org.prebid.server.proto.response.UsersyncInfo;
+import org.prebid.server.util.HttpUtil;
+
+public class UsersyncInfoBuilder {
```

Comment:
Just to think. Can this one be `UsersyncInfoResolver`? Then it is not required to use this complex and useless for this case builder approach(whole class will be refactored)?